### DISCUSSION TITLE
Is variable grid size (dx, dy, dz) possible?

### DISCUSSION THREAD

**Discussion by cooluser26** [epochpic/epoch#634] [Q&A]
Is it possible to change dx as a function of distance? For example, I have a spatially decaying electron profile and it would be nice if I can change dx to be a function of electron density at each point? Is there a strategy to simulate such profiles over long distances?

**Comment by Status-Mirror** [comment]
Hey @wnguyen1312,

EPOCH does not support multiple grid-sizes within the same simulation. There are no PIC codes which I am aware of that have this capability, but there may be a few out there somewhere.

Cheers,
Stuart
